docs/organizations/using-coding-standards.md

```diff
@@ -96,15 +122,15 @@ To edit an existing coding standard or change the repositories that follow that
 
     ![Editing an existing coding standard](images/coding-standard-edit.png)
 
-1.  Edit the current coding standard configurations and click the button **Next** to advance between the following pages:
+1.  Edit the current coding standard configurations and click the button **Next** to advance to the following pages:
```

Comment:
Maybe we could link to the advanced settings sections, as we moved lots of details into it.
Not sure if this is the best way, though...

```suggestion
1.  Edit the current coding standard configurations and click the button **Next** to advance to the following [configuration pages](#advanced-settings):
```